src/pages/hegel/reference/nothing/development.mdx

```diff
@@ -52,16 +53,61 @@ Hegel goes to add a link between the empty thinking (or empty intuiting) that fo
 
 Instead, it is better to understand Hegel's passage here to be an aid to the reader that merely reflects on the logic rather than developing it further. This being that the two categories appear to be equally devoid of any meaning, or map on to similar, if not same, manner of thinking. However, the claim that `being` is `nothing` is not on account of empty thinking or any other mental operation no, but purely through its concept.
```

Comment:
" but purely through its concept." - It might be worth spelling this out as "through its concept" might be a bit unclear.